packages/block-library/src/page-list/use-convert-to-navigation-links.js

```diff
@@ -30,11 +37,57 @@ export function convertToNavigationLinks( pages = [] ) {
 				// Use a placeholder if the child appears before parent in list.
 				linkMap[ parent ] = { innerBlocks: [] };
 			}
+			// Although these variables are not referenced, they are needed to store the innerBlocks in memory.
```

Comment:
I don't understand: how are they useful if nobody reads them?

Reply:
I don't 100% get it either. I think it means they are stored in memory, so that they don't get lost when a mutation happens. This is old code that might be possible to write in a clearer way, but I think that's outside of the scope of this PR.